src/components/status/PastIncidents.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { ArrowDown, ArrowUp } from "lucide-react";
+import { useState } from "react";
+
+interface Incident {
+  date: string;
+  report: string;
+}
+
+const incidents: Incident[] = [
+  {
+    date: "Aug 22, 2024",
+    report: "No incidents reported today.",
+  },
+  {
+    date: "Aug 21, 2024",
+    report: "No incidents reported.",
+  },
+  {
+    date: "Aug 20, 2024",
+    report: "Minor server outage from 2 PM to 3 PM.",
+  },
+  {
+    date: "Aug 19, 2024",
+    report: "Scheduled maintenance completed successfully.",
+  },
+  {
+    date: "Aug 18, 2024",
+    report: "System update applied, no issues reported.",
+  },
+];
+
+const PastIncidents: React.FC = () => {
```

Comment:
bad practice. please rectify


see [Here](https://dev.to/wdp/rethinking-typescript-practices-the-downside-of-reactfc-1ci7#:~:text=Implicit%20Definition%20of%20Children&text=FC%20%2C%20it%20implicitly%20includes%20children,by%20TypeScript%20due%20to%20React)

Reply:
Alright. Noted and implemented. Thank you